src/components/Vocal.js

```diff
@@ -115,6 +118,8 @@ const Vocal = ({
 		stopTimer()
 		stopRecognition()
 
+		triggerCommand(result)
```

Comment:
Correct me if I'm wrong, but I should be able to completely change the commands in use without suspending the speech recognition session, right?

I'm thinking of assisted typing use cases where I want to provide matching commands based on the current field / table cell.

Reply:
Like I said in the previous comment, that will be to consider as soon as the component handles continuous sessions